### ISSUE TITLE
Update function doesn't work

### ISSUE THREAD

**Issue by UnR34L** [nosmokingbandit/Watcher3#268]
Can't seem to update using update button. I haven't made any changes.

> ERROR [2018-08-06 16:18:29,778] core.version.execute_update.222: Update failed: error: Your local changes to the following files would be overwritten by merge:
> 	core/scheduler.py
> 	core/searchresults.py
> 	post scripts/deluge.py
> 	post scripts/downloadstation.py
> 	post scripts/nzbget.py
> 	post scripts/qbittorrent.py
> 	post scripts/rtorrent.py
> 	post scripts/sabnzbd.py
> 	post scripts/test.py
> 	post scripts/transmission.py
> Please commit your changes or stash them before you merge.
> Aborting
> Updating f600e1c..0b446ce
> ERROR [2018-08-06 16:18:29,778] core.ajax.update_server.452: Update Failed.

**Comment by UnR34L** [comment]
Once I stashed it worked fine, don't know why it happened in the first place.

**Comment by nosmokingbandit** [comment]
I once used SourceTree and it screwed up a lot of tracked files and other stuff. I thought I had gotten all of them, but I'm never using SourceTree again.
